packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/app/[lang]/layout.tsx

```diff
@@ -0,0 +1,73 @@
+import '@pantheon-systems/nextjs-kit/style.css';
+
```

Comment:
(very small picky thing) I'm starting to lean towards having 0 separators between imports, but I could be convinced otherwise. Thoughts? Either way I think the CSS imports should be grouped together.